include/seqan3/alignment/pairwise/detail/policy_optimum_tracker.hpp

```diff
@@ -66,6 +66,70 @@ struct max_score_updater
     }
 };
 
+/*!\brief A function object that compares and possibly updates the alignment optimum with the current cell.
+ * \ingroup pairwise_alignment
+ *
+ * \details
+ *
+ * Updates the current alignment optimum with the new score and the respective coordinate if the new score
+ * compares greater or equal to the score of the current optimum and the cell is either in the target row or column.
+ * For the banded alignment this additional check is needed to only track the cells that are really in the last
+ * row of the global alignment matrix and not at the end of the column that does not reach the last alignment row yet.
```

Comment:
```suggestion
 * For the banded alignment this additional check is needed because the band might not reach the last row of the matrix for all columns. 
 * Thus, the last cell of a column will only be checked if the corresponding coordinate points to the last row of the matrix.
```